apps/performance-tests/src/util/IModelUtilities.ts

```diff
@@ -3,11 +3,13 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import { TestIModelBuilder } from "presentation-test-utilities";
+import fs from "fs";
+import { getFullSchemaXml, TestIModelBuilder } from "presentation-test-utilities";
 import { IModelDb, SnapshotDb } from "@itwin/core-backend";
 import { BisCodeSpec, Code, ElementAspectProps, ElementProps, ModelProps } from "@itwin/core-common";
 
-export async function createIModel(name: string, localPath: string, cb: (builder: TestIModelBuilder) => void | Promise<void>) {
+export async function createIModel(name: string, localPath: string, cb: (builder: BackendTestIModelBuilder) => void | Promise<void>) {
+  await fs.promises.rm(localPath, { force: true });
```

Comment:
Is there a point to use promise api vs sync?

```suggestion
  fs.rmSync(localPath, { force: true });
```